src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/GlobalSearchService.kt

```diff
@@ -141,8 +144,7 @@ class GlobalSearchService(
   private fun setupIndexSearch(scroll: Scroll): SearchResponse {
 
     val searchSourceBuilder = SearchSourceBuilder().apply {
-      docValueField("prisonerNumber")
-      sort("prisonerNumber")
+      fetchSource(FetchSourceContext.DO_NOT_FETCH_SOURCE)
```

Comment:
Ive realised that prisonerNumber is actually also the document id, so no need to get any fields - may be much faster 🤞 . Does mean a manual sort is needed however.

Reply:
If you're getting the whole index up front maybe you could do a sort and *then* use Kotlin's minus to get the diffs? I love minus as you can tell 😄

Reply:
aww ... i cant let go of my lovely code ...... go on then! Ill do a minus